<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>798</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>685</width>
    <height>798</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_changes">
      <property name="minimumSize">
       <size>
        <width>656</width>
        <height>254</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>254</height>
       </size>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_clear">
         <property name="minimumSize">
          <size>
           <width>206</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>206</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_centre">
         <property name="minimumSize">
          <size>
           <width>285</width>
           <height>62</height>
          </size>
         </property>
         <property name="title">
          <string>Точка центра преобразования</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_centre_x">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_centre_x">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_centre_y">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_centre_y">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_rotate">
         <property name="minimumSize">
          <size>
           <width>206</width>
           <height>126</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>206</width>
           <height>126</height>
          </size>
         </property>
         <property name="title">
          <string>Поворот</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_rotate_degree">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>∠:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_x">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>22</height>
             </size>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_rotate">
            <property name="text">
             <string>Повернуть</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QGroupBox" name="groupBox_move">
         <property name="minimumSize">
          <size>
           <width>206</width>
           <height>126</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>206</width>
           <height>126</height>
          </size>
         </property>
         <property name="title">
          <string>Перенос</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_move">
            <property name="text">
             <string>Перенести</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_move_y">
            <property name="text">
             <string>dY:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_move_x">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>22</height>
             </size>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_move_y">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>22</height>
             </size>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_move_x">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>dX:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_scale">
         <property name="minimumSize">
          <size>
           <width>206</width>
           <height>126</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>206</width>
           <height>126</height>
          </size>
         </property>
         <property name="title">
          <string>Масштабирование</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_scale_x">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>kX:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_scale_x">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>22</height>
             </size>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_scale_y">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>kY:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_scale_y">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>22</height>
             </size>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_scale">
            <property name="text">
             <string>Масштабировать</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_undo">
         <property name="minimumSize">
          <size>
           <width>206</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>206</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Отменить действие</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_autoscale">
         <property name="minimumSize">
          <size>
           <width>206</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>206</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Масштабировать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="canvas_widget" native="true"/>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Инфо</string>
    </property>
    <addaction name="about_proramm"/>
    <addaction name="about_author"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="about_proramm">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="about_author">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
